### PR TITLE
Starter Kit installation/setup

### PR DESCRIPTION
**Pull request by keharper**
## Purpose of this pull request

This pull request (PR) adds installation/setup/deployment information.

## Affected pages

- ...

## Links to Magento Open Source code

- ...

### CODE REVIEW

File: src/pages/app-development/starter-kit/project-setup.md [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+
+If you are running Adobe Commerce 2.4.4 or 2.4.5, you must install modules to enable eventing, as described in [Install Adobe I/O Events for Adobe Commerce](../../events/installation.md). These modules are installed automatically in subsequent versions of Commerce.
+
+## Starter Kit set-up and onboarding
+
+You are now ready to download the Adobe Commerce Starter Kit and set up your development.
+
+### Download and configure the project
+
+<!-- Note to reviewer: Will the enterprise-integration/starter-kit repo be mirrored? It looks like a private repo -->
```
Comment (rojoangel) [inline src/pages/app-development/starter-kit/project-setup.md:109]: During this beta phase, we are distributing the starter kit as a zip version of the git repo.
Once the beta phase is completed, the intention is to make the starter kit available in the marketplace.

File: src/pages/app-development/starter-kit/project-setup.md [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+```
+
+You can confirm the success of the deployment in the Adobe Developer Console by navigating to the Runtime section on your workspace.
+
+![Adobe I/O Runtime actions](../../_images/starterkit/runtime-actions.png)
+
+## Onboarding
+
+The onboarding process configuring event registrations and completing the eventing configuration in Adobe Commerce.
+
```
Comment (rojoangel) [inline src/pages/app-development/starter-kit/project-setup.md:187]: Shouldn't present tense be used here?
```suggestion
The onboarding process configures event registrations and completes the eventing configuration in Adobe Commerce.

```

File: src/pages/app-development/starter-kit/project-setup.md [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+### Configure the event registrations
+
+By default, the `./onboarding/custom/registrations.json` config file creates all the registrations for all entities that are prtesen in the repo's `app.config.yaml` file. You can edit the `registrations.json` file to remove any Commerce or back office unneeded registrations. For example, the YAML file shown in the [Configure the project](#configure-the-project) section commented out the `product-backoffice` package. In this case, you must remove backoffice from the product entity:
```
Comment (rojoangel) [inline src/pages/app-development/starter-kit/project-setup.md:190]: Typo?
```suggestion
By default, the `./onboarding/custom/registrations.json` config file creates all the registrations for all entities that are present in the repo's `app.config.yaml` file. You can edit the `registrations.json` file to remove any Commerce or back office unneeded registrations. For example, the YAML file shown in the [Configure the project](#configure-the-project) section commented out the `product-backoffice` package. In this case, you must remove backoffice from the product entity:
```

**Review by rojoangel — APPROVED** [review]
Looks good to me. Nice job!
One thing that concerns me is the `src/pages/app-development/project-setup.md` page. It seems to be more related to general App Builder application development than specifically to the starter kit... I just want to be sure that we are not going to confuse starter-kit users.

File: src/pages/app-development/starter-kit/project-setup.md [hunk 4]
```diff
@@ -262,7 +262,7 @@ You must configure Commerce to communicate with your project. Configuration incl
 
 ### Subscribe to events in Adobe Commerce
 
-Use the `bin/magento events:subscribe` command to subscribe to events, as described in [Subscribe and register events](https://developer.adobe.com/commerce/extensibility/events/configure-commerce/#subscribe-and-register-events). The following table defines the events for each supported entity that you must subscribe to and lists the required fields, if applicable.
+Use the `bin/magento events:subscribe` command to subscribe to events, as described in [Subscribe and register events](../../events/configure-commerce.md#subscribe-and-register-events). The following table defines the events for each supported entity that you must subscribe to and lists the required fields, if applicable.
```
Comment (danidelcar) [inline src/pages/app-development/starter-kit/project-setup.md:265]:
```
Use the `bin/magento events:subscribe` command to subscribe to events, as described in [Subscribe and register events](../../configure-commerce.md#subscribe-and-register-events). The following table defines the events for each supported entity that you must subscribe to and lists the required fields, if applicable.
```

File: src/pages/app-development/starter-kit/project-setup.md [hunk 5]
```diff
@@ -0,0 +1,276 @@
...
+
+1. On the next page, click the **Allow** button to display the **Integration Tokens for Extensions** page.
+
+   ![Integration tokens](../../_images/starterkit/secrets.png)
+
+   You will need the integration details (consumer key, consumer secret, access token, and access token secret) to configure the Starter Kit. Copy these values to a safe place and click **Done**.
+
+## Install Adobe I/O Events for Adobe Commerce (Commerce 2.4.4 and 2.4.5 only)
+
+If you are running Adobe Commerce 2.4.4 or 2.4.5, you must install modules to enable eventing, as described in [Install Adobe I/O Events for Adobe Commerce](../../events/installation.md). These modules are installed automatically in subsequent versions of Commerce.
```
Comment (jeff-matthews) [inline src/pages/app-development/starter-kit/project-setup.md:101]:
```suggestion
If you are running Adobe Commerce 2.4.4 or 2.4.5, you must install modules to enable eventing, as described in [Install Adobe I/O Events for Adobe Commerce](../events/installation.md). These modules are installed automatically in subsequent versions of Commerce.
```

File: src/pages/app-development/starter-kit/project-setup.md (hunk 4 above)
Comment (danidelcar) [reply src/pages/app-development/starter-kit/project-setup.md:265]: remove the `events/`

**Review by jeff-matthews — COMMENTED** [review]
~Try my suggestion to fix the link check error @keharper~

Disregard this comment

File: src/pages/app-development/starter-kit/project-setup.md (hunk 4 above)
Comment (danidelcar) [reply src/pages/app-development/starter-kit/project-setup.md:265]: or `[Subscribe and register events](../events/configure-commerce.md#subscribe-and-register-events)`

File: src/pages/app-development/starter-kit/project-setup.md [hunk 6]
```diff
@@ -0,0 +1,272 @@
...
+  {
+    key: 'backoffice',
+    id: '<Backoffice Provider ID>',
+    instanceId: '<Instance ID of backoffice provider',
+    label: 'Backoffice Provider'
+  }
+]
+```
+
+Check your App developer console to confirm the registrations were created.
```
Comment (jhadobe) [inline src/pages/app-development/starter-kit/project-setup.md:225]:
```suggestion
Check your App in the Developer Console to confirm the registrations were created.
```

File: src/pages/app-development/starter-kit/project-setup.md [hunk 7]
```diff
@@ -0,0 +1,272 @@
...
+
+   **Note**: You must enable cron so that Commerce can send events to the endpoint.
+
+1. Enter the merchant's company name in the **Merchant ID** field. You must use alphanumeric and underscores only.
+
+1. In the **Environment ID** field, enter a temporary name for your workspaces while in development mode. When you are ready for production, change this value to a permanent value, such as Production.
+
+1. (Optional) By default, if an error occurs when Adobe Commerce attempts to send an event to Adobe I/O, Commerce retries a maximum of seven times. To change this value, uncheck the Use system value checkbox and set a new value in the **Maximum retries to send events** field.
+
+1. (Optional) By default, Adobe Commerce runs a cron job (clean_event_data) every 24 hours that delete event data three days old. To change the number of days to retain event data, uncheck the Use system value checkbox and set a new value in the Event retention time (in days) field.
```
Comment (jhadobe) [inline src/pages/app-development/starter-kit/project-setup.md:255]:
```suggestion
1. (Optional) By default, Adobe Commerce runs a cron job (clean_event_data) every 24 hours that deletes event data three days old. To change the number of days to retain event data, uncheck the **Use system value** checkbox and set a new value in the **Event retention time (in days)** field.
```

File: src/pages/app-development/starter-kit/project-setup.md [hunk 8]
```diff
@@ -0,0 +1,272 @@
...
+
+1. Enable Commerce Eventing by setting **Enabled** to `Yes`.
+
+   **Note**: You must enable cron so that Commerce can send events to the endpoint.
+
+1. Enter the merchant's company name in the **Merchant ID** field. You must use alphanumeric and underscores only.
+
+1. In the **Environment ID** field, enter a temporary name for your workspaces while in development mode. When you are ready for production, change this value to a permanent value, such as Production.
+
+1. (Optional) By default, if an error occurs when Adobe Commerce attempts to send an event to Adobe I/O, Commerce retries a maximum of seven times. To change this value, uncheck the Use system value checkbox and set a new value in the **Maximum retries to send events** field.
```
Comment (jhadobe) [inline src/pages/app-development/starter-kit/project-setup.md:253]:
```suggestion
1. (Optional) By default, if an error occurs when Adobe Commerce attempts to send an event to Adobe I/O, Commerce retries a maximum of seven times. To change this value, uncheck the **Use system value** checkbox and set a new value in the **Maximum retries to send events** field.
```

File: src/pages/app-development/starter-kit/project-setup.md [hunk 9]
```diff
@@ -0,0 +1,272 @@
...
+Entity | Event | Required fields
+--- | --- | ---
+Product | `com.adobe.commerce.observer.catalog_product_delete_commit_after` | Not applicable
+Product | `com.adobe.commerce.observer.catalog_product_save_commit_after`   | `created_at`, `updated_at`
+Customer | `com.adobe.commerce.observer.customer_save_commit_after` | `created_at`, `updated_at`
+Customer | `com.adobe.commerce.observer.customer_delete_commit_after` | Not applicable
+Customer group | `com.adobe.commerce.observer.customer_group_save_commit_after` | Not applicable
+Customer group | `com.adobe.commerce.observer.customer_group_delete_commit_after` | `customer_group_code`
+Order | `com.adobe.commerce.observer.sales_order_save_commit_after` | `created_at`, `updated_at`
+Stock | `com.adobe.commerce.observer.cataloginventory_stock_item_save_commit_after` | Not applicable
```
Comment (jhadobe) [inline src/pages/app-development/starter-kit/project-setup.md:272]: Shipment event seems to be missing?

**Review by jhadobe — APPROVED** [review]

**Pull request merged**